### ISSUE TITLE
Bug: sbb-usermenu shifts username on clicking

### ISSUE THREAD

**Issue by jeripeierSBB** [SchweizerischeBundesbahnen/sbb-angular#124]
**Describe the bug**
if a user is logged in an clicks on his username (sbb-usermenu) (to expand the dropdown), the avatar/initials and the name is shifted.

**To Reproduce**
1. Go to https://sbb-angular.app.sbb.ch/latest/content/usermenu
2. Click on Login
3. Click on username
4. username and avatar are shifted instead of staying at its place

**Expected behavior**
avatar and username stay at its place

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome

**Additional context**
We "solved" the problem by appending following styles inside our application:

```
:host {
  ::ng-deep {
    sbb-usermenu {
      .sbb-usermenu-logged-in {

        /*
          if username is shown and dropdown collapsed -> correct margins of box
         */
        &.sbb-dropdown-trigger:not(.sbb-usermenu-logged-in-active) .sbb-usermenu-logged-in-collapsed {
          border-left: 2px solid transparent;
          margin-top: 2px;
        }

        /*
          if username is shown and dropdown collapsed -> correct margin of user name
        */
        .sbb-usermenu-logged-in-collapsed:not(.sbb-usermenu-logged-in-collapsed-dropdown-expanded) .sbb-usermenu-logged-in-expanded-info-user .sbb-usermenu-logged-in-expanded-displayName {
          margin-top: -2px;
        }

        /*
          if username is shown and dropdown is expanded -> shift up user name and icon
         */
        .sbb-usermenu-logged-in-collapsed-dropdown-expanded {
          margin-top: 0;
        }
      }
    }
  }
}
```

**Comment by kyubisation** [comment]
Closed by #128

**Comment by jeripeierSBB** [comment]
@kyubisation For me this bug is still present, see video: ![ezgif-4-182d05c07191](https://user-images.githubusercontent.com/29278232/64966606-648ef180-d89f-11e9-8870-64af78da046f.gif)
